### ISSUE TITLE
dockerhub crystallang/crystal:0.19.2 does not ship with yaml

### ISSUE THREAD

**Issue by grosser** [crystal-lang/crystal#3347]
```
error while loading shared libraries: libyaml-0.so.2: cannot open shared object file: No such file or directory
```

is this by design ? ... can we have a nicer error message then ?
... if it's not by design ... any idea how to fix ?

**Comment by raydf** [comment]
Something like:

In ubuntu:
```bash
sudo apt-get install libyaml-dev
```

**Comment by asterite** [comment]
Maybe docker images should come with everything the standard library uses...

**Comment by grosser** [comment]
sounds reasonable to me ...

**Comment by luislavena** [comment]
Hello,

This has been solved by #3441:

```
$ docker run -it crystallang/crystal bash

root@a873bceeab61:/opt/crystal# crystal --version
Crystal 0.20.3 [b1416e2] (2016-12-23)

root@a873bceeab61:/opt/crystal# cd /tmp/

root@a873bceeab61:/tmp# cat test.cr 
require "yaml"

puts YAML.dump({:foo => 10})

root@a873bceeab61:/tmp# crystal build test.cr

root@a873bceeab61:/tmp# ./test 
---
foo: 10

root@a873bceeab61:/tmp# ldd test
	linux-vdso.so.1 =>  (0x00007fffe47d3000)
	libyaml-0.so.2 => /usr/lib/x86_64-linux-gnu/libyaml-0.so.2 (0x00007f7997a2e000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f7997810000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f7997607000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f7997403000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f79971ed000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f7996e27000)
	/lib64/ld-linux-x86-64.so.2 (0x000055e19d47d000)
```

Cheers!
